### DISCUSSION TITLE
git lfs bandwidth not initialized

### DISCUSSION THREAD

**Discussion by designsh** [git-lfs/git-lfs#5076] [Q&A]
The bandwidth reset deadline has passed, but the bandwidth itself is not reset. Is there any solution other than payment
![bandwidth](https://user-images.githubusercontent.com/23091061/178744721-37742241-bdd7-4f94-9098-772ed083185d.png)
?

**Comment by bk2204** [comment]
Hey,

It looks like your question is about GitHub, but this project is only for the open source Git LFS project.  You'd want to contact GitHub Support about this issue.

**Comment by Uvais5** [comment]
i face same problem can you please tell me how to fix this

**Reply by bk2204** [reply]
As I mentioned above, this repository is for dealing with the Git LFS open source project.  If you have a problem with GitHub, you should contact the GitHub Support team, who can help you get this fixed.

It's fine to ask questions generally about how you'd use Git LFS with GitHub or other hosting platforms, but billing and usage are not involved in the open source project in any way and need to be addressed with the hosting platform directly.

**Reply by Uvais5** [reply]
i contact github support but not reply  from github

**Reply by bk2204** [reply]
It may take them a little while to get back to you.  There's nothing that can be done by posting here, so you'll have to wait until they respond.

**Reply by Uvais5** [reply]
ok buddy  thanks for helping me
